pkg/acsengine/template_generator.go

```diff
@@ -539,6 +542,48 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 			// return the custom data
 			return fmt.Sprintf("\"customData\": \"[base64(concat('%s'))]\",", str)
 		},
+		"GetKubernetesMasterCustomDataVMSS": func(profile *api.Properties) string {
```

Comment:
There is a code redundancy with GetKuberneetesMasterCustomData here. Can we move this to code to another method that will be used by both MasterCustomData as well as MasterCustomDataVMSS?